frontend/javascripts/oxalis/shaders/filtering.glsl.ts

```diff
@@ -113,14 +113,18 @@ export const getMaybeFilteredColorOrFallback: ShaderModule = {
       vec4 fallbackColor,
       bool supportsPrecomputedBucketAddress
     ) {
+      vec4[2] returnValue;
       vec4 color = getMaybeFilteredColor(layerIndex, d_texture_width, packingDegree, worldPositionUVW, suppressBilinearFiltering, supportsPrecomputedBucketAddress);
 
+      vec4 usedFallbackColor = vec4(0.0);
```

Comment:
the semantics of this var is a simple boolean, right? in that case, maybe define a struct with a vec4 and a bool?